modules/gapi/src/api/gmat.cpp

```diff
@@ -36,6 +36,39 @@ const cv::GOrigin& cv::GMat::priv() const
     return *m_priv;
 }
 
+namespace {
+std::vector<int> checkVectorImpl(const int width, const int height, const int chan, const int n)
```

Comment:
> namespace {

may be just "static"?

(to avoid unnecessary obfuscated names in debugger)

Reply:
Agree. Thanks! Changed